packages/components/pie-toast/src/index.ts

```diff
@@ -219,30 +236,78 @@ export class PieToast extends RtlMixin(LitElement) implements ToastProps {
         dispatchCustomEvent(this, ON_TOAST_CLOSE_EVENT, { targetNotification: this });
     }
 
+    /**
+     * Util method that returns an icon from a variant that has default icon.
+     *
+     * @private
+     */
+    private getVariantIcon () {
+        switch (this.variant) {
+            case 'info':
+                return html`<icon-info-circle class="c-toast-icon" size="s" data-test-id="${componentSelector}-heading-icon-info"></icon-info-circle>`;
```

Comment:
Could use `componentClass` here:

```suggestion
                return html`<icon-info-circle class="${componentClass}-icon" size="s" data-test-id="${componentSelector}-heading-icon-info"></icon-info-circle>`;
```

Reply:
Ops! 😅